GitCommands/Git/GitModule.cs

```diff
@@ -60,7 +60,7 @@ public ConflictedFileData(string hash, string filename)
         public string Filename;
     }
 
-    [DebuggerDisplay("{Filename}")]
+    [DebuggerDisplay("{" + nameof(Filename) + "}")]
```

Comment:
Is this the same? (Question, I am not familiar)

Reply:
should be the same

Reply:
Yes it's the same. The `DebuggerDisplayAttribute` evaluates a format string where member names are enclosed in braces, and shows the resulting string in the debugger for instances of the associated type.

By using `nameof` here, we are protected against breakages from automated refactoring tools that cannot introspect into the original format string.

Reply:
Can this be `[DebuggerDisplay($"{nameof(Filename)}")]`

Reply:
No. It can be 

``[DebuggerDisplay($"{{{nameof(Filename)}}}")]``

Interpolated strings escape {} with double {}

Reply:
@vbjay you can't use `FormattableString` in attributes. Attribute arguments must be compile time constants. Even though the compiler could compute such a compile time value here, it doesn't. If you want to use `nameof`, the approach I have gone for is the only working route I know of.

Reply:
Oops. Good point. I just saw the text wrong format of the interpolated
string.

Reply:
Oops. Good point. I just saw the text wrong format of the interpolated
string.

Reply:
No worries then, I'm fine with that.

Reply:
➡️ Related to dotnet/csharplang#384